lms/djangoapps/verify_student/tests/test_services.py

```diff
@@ -25,20 +28,22 @@ def setUp(self):
         super(TestReverificationService, self).setUp()
 
         self.user = UserFactory.create(username="rusty", password="test")
-        course = CourseFactory.create(org='Robot', number='999', display_name='Test Course')
-        self.course_key = course.id
```

Comment:
I have changed this as it is a id of the course and course_key is later generated using `CourseKey` module.